lib/internal/test_runner/test.js

```diff
@@ -470,10 +469,7 @@ class Suite extends Test {
     this.parent.activeSubtests++;
     this.startTime = hrtime();
     const subtests = this.skipped ? [] : this.subtests;
-    await ArrayPrototypeReduce(subtests, async (prev, subtest) => {
-      await prev;
-      await subtest.run();
-    }, PromiseResolve());
+    await PromiseAll(ArrayPrototypeMap(subtests, (subtests) => subtests.start()));
```

Comment:
I think it makes sense to use `SafePromiseAll` here in case the user test cases mutate e.g. `Array.prototype[Symbol.iterator]` or `Promise.prototype.then`.

```suggestion
    await SafePromiseAll(subtests, (subtests) => subtests.start());
```

Reply:
@aduh95 `SafePromiseAll` only appears in the primordials docs, I was not able to find an implementation for it, importing it returns `undefined`.
what am I missing? is this something you want me to implement?

Reply:
just found it 😂  https://github.com/nodejs/node/pull/43728

Reply:
Yes sorry, I commented right after it got merged, I should have mentioned that a rebase was necessary x)